step-info.yml

```diff
@@ -0,0 +1 @@
+maintainer: community
```

Comment:
please move this file to this path in this PR: `steps/perfecto-upload/step-info.yml`
currently it is in the root of the repo, that is why the CI failed.

Reply:
Have done.